components/RedirectionBanner.vue

```diff
@@ -14,21 +14,15 @@
       >
         <h2>{{ header }}</h2>
         <p class="text-sm">{{ subheader }}</p>
-        <nuxt-link
-          :to="cta.href"
-          class="mt-5 flex"
-          :class="{
-            'bg-blue-200 text-white px-5 py-2 rounded-md': cta.isButton,
-          }"
+        <BaseButton
+          :arrow="cta.arrow"
+          :variant="cta.variant"
+          class="mt-5"
```

Comment:
Where is the `href` property, or it's redundant right now? Cause in deleted `nuxt-link` there is a `cta.href`

Reply:
By the way, what is the `cta` meaning? 🤔

Reply:
@WawrzenskiPatryk `call to action`
@elmir-ismayilzada You are correct atttibutes `href` and `to` were missing

Reply:
@WawrzenskiPatryk what is your verdict on 'cta' issue?

Reply:
We can leave `cta` for now but I think if anyone else got confused by it we should change it to something like `button-props`